rps.js

```diff
@@ -0,0 +1,21 @@
+// JS Functions HW  , Q1 
+// Rock, Paper, Scissors (watch the game)
+
+function random(str) {
+    return str[ Math.floor(Math.random()*str.length) ];
+}
+
+
+function playGame( userPicked ) {
+    let computerPicked = [ 'rock', 'paper', 'scissors' ];
+    let userScore = [ 'win', 'lose', 'tie' ];
```

Comment:
the only outcome that comes is a string.

add the logic for the game so that there is a winner and a loser and if there is a tie